<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsWFSSourceSelectBase</class>
 <widget class="QDialog" name="QgsWFSSourceSelectBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add WFS Layer from a Server</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="GroupBox1">
     <property name="title">
      <string>Server connections</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QComboBox" name="cmbConnections"/>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btnConnect">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>C&amp;onnect</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnNew">
          <property name="text">
           <string>&amp;New</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Edit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnDelete">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>171</width>
            <height>30</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnLoad">
          <property name="toolTip">
           <string>Load connections from file</string>
          </property>
          <property name="text">
           <string>Load</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSave">
          <property name="toolTip">
           <string>Save connections to file</string>
          </property>
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTreeWidget" name="treeWidget">
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="columnCount">
      <number>3</number>
     </property>
     <column>
      <property name="text">
       <string>Title</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Abstract</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="gbCRS">
     <property name="title">
      <string>Coordinate reference system</string>
     </property>
     <layout class="QHBoxLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="margin">
       <number>9</number>
      </property>
      <item>
       <widget class="QLabel" name="labelCoordRefSys">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>441</width>
          <height>23</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnChangeSpatialRefSys">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Change ...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="mBboxCheckBox">
     <property name="text">
      <string>Only request features overlapping the current view extent</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cmbConnections</tabstop>
  <tabstop>btnConnect</tabstop>
  <tabstop>btnNew</tabstop>
  <tabstop>btnEdit</tabstop>
  <tabstop>btnDelete</tabstop>
  <tabstop>treeWidget</tabstop>
  <tabstop>btnChangeSpatialRefSys</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
